crates/shared/src/consts.rs

```diff
@@ -1 +1 @@
-pub const EXPECTED_RPC_VERSION: &str = "0.6.0";
+pub const EXPECTED_RPC_VERSION: &str = "0.7.0-rc2";
```

Comment:
nitpick: I think a final version of 0.7.0 was released today, it does not seem to contain any changes to rc, but still we should probably consider updating when possible

Reply:
https://github.com/foundry-rs/starknet-foundry/issues/1890